src/marlin_stubs/host/M115.cpp

```diff
@@ -221,6 +221,9 @@ void GcodeSuite::M115() {
     #if HAS_HEATED_CHAMBER
                  ,
         true
+    #elif HAS_CHAMBER_API()
+                 ,
+        true
```

Comment:
You gotta check chamber capabilities. Just reporting true if HAS_CHAMBER_API() won't work.